m0t1fs/linux_kernel/st/m0t1fs_rpc_cancel_test.sh

```diff
@@ -231,21 +231,21 @@ rcancel_issue_writes_n_cancel()
 
 	echo "Test: Creating $wc_nr_files files on m0t1fs"
 	for ((i=0; i<$wc_nr_files; ++i)); do
-		touch $wc_file_base$i || break
+		touch "$wc_file_base""$i" || break
```

Comment:
Do you think the following is better?
```
touch "$wc_file_base$i" || break
```

This applies to all occurrences in this patch

Reply:
done